src/cli/main.rs

```diff
@@ -403,12 +407,31 @@ fn format(opt: opt::Opt) -> Result<i32> {
                         None => false,
                     };
 
+                    #[cfg(feature = "editorconfig")]
+                    let path = is_default_config.then(|| {
+                        match &opt.stdin_filepath {
+                            Some(filepath) => filepath.to_path_buf(),
+                            // HACK: this is not a real glob
+                            None => cwd.join("*.lua"),
+                        }
+                    });
+
                     pool.execute(move || {
+                        #[cfg(not(feature = "editorconfig"))]
+                        let c = config;
+                        #[cfg(feature = "editorconfig")]
+                        let c = path
+                            .and_then(|path| {
+                                editorconfig::parse(None, &path)
+                                    .expect("could not parse editorconfig")
+                            })
+                            .unwrap_or(config);
+
```

Comment:
I was not able to use `cwd` after `move` (`use occurs due to use in closure`), maybe there is an obvious fix.

Reply:
Ah fair. I'm not sure from first glance without playing around to try and please the compiler.

Do we actually even need the cwd here? Could the glob `*.lua` be sufficient? Or might that miss some config

Reply:
Good point: `cwd` was not needed, problem solved.

Note that `*.lua` is **not** a glob, `a.lua` could be used instead: it would also match `[*.lua]` (the just core library expects a file path).

If `[*.luau]` or another extension is defined in `.editorconfig`, it won't be taken into account when formatting from stdin without `--stdin-filepath`.

Reply:
Fair enough. I think its reasonable to expect the user to provide stdin filepath in most cases (typically stdin is used for editor integrations or scripts)